pom.xml

```diff
@@ -145,4 +158,4 @@
       </plugin>
     </plugins>
   </build>
-</project>  
```

Comment:
Please avoid unrelated changes. They decrease the PR readability and also may cause merge conflicts